CMakeLists.txt

```diff
@@ -23,9 +23,17 @@ include(PandoraCMakeSettings)
 include_directories(include)
 
 find_package(PandoraSDK 03.04.01 REQUIRED)
-find_package(LArContent 03.19.03 REQUIRED)
 
-foreach(package PandoraSDK LArContent)
+set(LAR_CONTENT_LIBRARY_NAME "LArContent")
+find_package(${LAR_CONTENT_LIBRARY_NAME} 03.19.03 REQUIRED)
```

Comment:
Note/reminder: presumably this version number will need to be reviewed and updated at the point of merging?

Reply:
Indeed, despite the branch name I've held off for now pending confirmation of which version this is going to end up being.